suse-x64/Dockerfile

```diff
@@ -157,6 +158,28 @@ RUN chmod +x /build-gcc.sh \
     && /build-gcc.sh \
     && rm /build-gcc.sh
 
+RUN curl -sL -o /tmp/rpm-4.14.3.tar.bz2 http://ftp.rpm.org/releases/rpm-4.14.x/rpm-4.14.3.tar.bz2 \
+    && cd /tmp \
+    && echo "13711268181a6e201eb9d4eb06384a7fcc42dcc6c9057a62c53472cfc5ba44b5  rpm-4.14.3.tar.bz2" | sha256sum --check \
+    && cp /usr/include/db4/db.h /usr/include/db.h \
+    && tar -xjf /tmp/rpm-4.14.3.tar.bz2 \
+    && cd rpm-4.14.3 \
+    && curl -sL -o db-4.5.20.tar.gz http://download.oracle.com/berkeley-db/db-4.5.20.tar.gz \
+    && echo "f52cd5cea899823dd200d56556f70b33c55e48a33bb7b65ee128968dc10ca82d  db-4.5.20.tar.gz" | sha256sum --check \
+    && tar xzf db-4.5.20.tar.gz \
+    && ln -s db-4.5.20 db \
+    && cd db/build_unix/ \
```

Comment:
Why are we downloading another db4 here? I don't see it installed/used anywhere

Reply:
It's needed for RPM to build he access his db with the ``db`` symlink. To get more info you can look at the INSTALL file in the source

Reply:
From what I understood in the `INSTALL` file, you *either* use an embedded BDB (for which you'd need to download) *or* use a system installation of it (`libdb4_8`, this would probably be preferable in this case).

Reply:
Ok yea it makes sense